utils/generate-examples.py

```diff
@@ -143,6 +143,17 @@
     "modules/cluster/remote-clusters-migration.asciidoc",
     "modules/cluster/remote-clusters-troubleshooting.asciidoc",
     "upgrade/archived-settings.asciidoc",
+    "index-modules/allocation/filtering.asciidoc",
+    "index-modules/allocation/delayed.asciidoc",
+    "index-modules/allocation/prioritization.asciidoc",
+    "index-modules/allocation/total_shards.asciidoc",
+    "index-modules/allocation/data_tier_allocation.asciidoc",
+    "index-modules/blocks.asciidoc",
+    "index-modules/similarity.asciidoc",
+    "index-modules/slowlog.asciidoc",
+    "index-modules/store.asciidoc",
+    "index-modules/index-sorting.asciidoc",
+    "index-modules/indexing-pressure.asciidoc"
```

Comment:
Would you mind adding a trailing comma here to fix the lint!

Reply:
I hope you had the best time off! I have been careful to only add the examples I have investigated, but noticed a number for pre-existing files that look to be making some changes to their examples, I was going to see if I pushed a draft if we could review them and confirm that they should be added, but wanted to get a feel for it this is over stepping.

Reply:
Hey Annie, thanks for the careful consideration here. This is not over stepping, but maybe unnecessary work as we update existing examples just before releases anyway.

Reply:
What would be nice however would be to modify utils/generate-examples.py to exclude empty examples.

Reply:
I know I way behind on this but would love to take a whack at modifying utils/generate-examples.py to exclude empty examples, if you haven't already?